pkg/util/config/config.go

```diff
@@ -78,6 +78,7 @@ type Config struct {
 	DbHostedZone                string `name:"db_hosted_zone" default:"db.example.com"`
 	EtcdScope                   string `name:"etcd_scope" default:"service"`
 	WALES3Bucket                string `name:"wal_s3_bucket"`
+	PgDailyLogS3Bucket          string `name:"pg_daily_log_s3_bucket"`
```

Comment:
just rename to "log_s3_bucket"? other parts of the variable are misleading or not required.